content/en/Platform Deep Dive/Credits/ptaas-tiers.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Cobalt PtaaS Tiers"
+linkTitle: "PtaaS Tiers"
+weight: 20
+description: >
+  Learn more about our pricing packages, also known as _PtaaS tiers_.
+---
+
+{{% pageinfo %}}
+We offer three [PtaaS tiers](https://www.cobalt.io/pentest-pricing) to best suit your budget and testing goals.
+{{% /pageinfo %}}
+
+To get started, check our [pricing model](https://www.cobalt.io/pentest-pricing), and select a plan that matches your testing expectations and business needs.
+
+| Standard | Premium | Enterprise |
+|---|---|---|
+| For teams in need of a speedy, annual pentest to meet a compliance need or client request | For teams looking to build a structured pentest program to meet compliance needs and improve overall security | For teams looking to scale their pentest programs to meet compliance needs, increase testing frequency, and improve overall security |
+
+## Offering Details
+
+The availability of some features in the Cobalt platform depends on the {{% ptaas-tier %}} that your organization purchased. Refer to the table below to learn more about our offerings.
+
+| Offering | Tier | Description |
```

Comment:
I appreciate the work that you've done with this table. But shouldn't we use https://www.cobalt.io/pentest-pricing as the SSoT for this content?

Reply:
Good explanation. Approving this PR.